src/content/changes/highlights/2021-12-17.md

```diff
@@ -0,0 +1,15 @@
+---
+date: 2021-12-17T10:00:00
+title: Highlights for the week ending December 17, 2021
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Apps
+
+- [aws-load-balancer-controller-app](https://github.com/giantswarm/aws-load-balancer-controller-app) [v1.0.2](https://github.com/giantswarm/aws-load-balancer-controller-app/blob/main/CHANGELOG.md#102---2021-12-14) improves labels for monitoring.
+- [cert-manager-app](https://github.com/giantswarm/cert-manager-app/) [v2.12.0](https://github.com/giantswarm/cert-manager-app/blob/master/CHANGELOG.md#2120---2021-12-16) upgrades the upstream image to [`v1.6.1`](https://github.com/jetstack/cert-manager/releases/tag/v1.6.1). This version stops serving cert-manager API versions `v1alpha2, v1alpha3, and v1beta1`. If you need to upgrade your resources, [this document](https://cert-manager.io/docs/installation/upgrading/remove-deprecated-apis/#upgrading-existing-cert-manager-resources) explains the process.
+- [external-dns-app](https://github.com/giantswarm/external-dns-app/) [v2.7.0](https://github.com/giantswarm/external-dns-app/blob/master/CHANGELOG.md#270---2021-12-16) upgrades to upstream [v0.10.2](https://github.com/kubernetes-sigs/external-dns/releases/tag/v0.10.2). This version removes support for Kubernetes versions <= 1.18.
+- [kong-app](https://github.com/giantswarm/kong-app/) [v2.3.5](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#235---2021-12-14) improves labels for monitoring.
```

Comment:
I think you can add the whole EFK / Grafana and Prometheus Operator things here :)